projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -1439,6 +1439,51 @@ protected void SetPlurality(int? value)
                 Subject.MultipleBirth.Extension.Add(plurality);
             }
         }
+    
+        /// <summary>Patient Fetal Death</summary>
+        /// <value></value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.PatientFetalDeath = true;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Patient Fetal Death: {ExampleBirthRecord.PatientFetalDeath}");</para>
+        /// </example>
+        [Property("PatientFetalDeath", Property.Types.Bool, "PatientFetalDeath", "PatientFetalDeath", false, VR.IGURL.PatientFetalDeath, true, 288)]
+        [FHIRPath("Bundle.entry.resource.where($this is Patient)", "deceased")]
+
+        public bool? PatientFetalDeath
+        {
+            get
+            {
+                if (Subject != null && Subject.Deceased != null)
+                {
+                    Extension patientFetalDeath = Subject.Deceased.Extension.FirstOrDefault(ext => ext.Url == VRExtensionURLs.PatientFetalDeath);
+                    if (patientFetalDeath != null)
+                    {
+                        bool? fetalDeath = ((FhirBoolean)patientFetalDeath.Value).Value;
+                        return fetalDeath;
+                    }
+                }
+                return null;
+            }
+            set
+            {
+                if (Subject.Deceased == null)
+                {
+                    Subject.Deceased = new FhirBoolean(true);
```

Comment:
from reading the IG, it says Required Pattern: true for both "deceased" and the value extension. Does that mean that this field should always be set to true be default for a Fetal Death Record? And also, that this field would never be set to false?

Reply:
https://build.fhir.org/ig/HL7/fhir-bfdr/StructureDefinition-Patient-decedent-fetus.html I was looking at the Key Elements table here.

Reply:
For a fetal death record, it should always be set to true. For a birth record, it can either be False or null.